pkg/client/core/service_argocd_client.go

```diff
@@ -28,6 +28,53 @@ type argoCDService struct {
 	param    client.ParameterService
 }
 
+func (s *argoCDService) DeleteArgoCD(ctx context.Context, opts client.DeleteArgoCDOpts) error {
+	err := s.spinner.Start("argocd")
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		err = s.spinner.Stop()
+	}()
+
+	info := s.state.GetArgoCD(opts.ID)
+
+	err = s.manifest.DeleteNamespace(ctx, api.DeleteNamespaceOpts{
+		ID:        opts.ID,
+		Namespace: "argocd",
+	})
+	if err != nil {
+		return err
+	}
+
+	err = s.cert.DeleteCognitoCertificate(ctx, api.DeleteCognitoCertificateOpts{
```

Comment:
Should we delete Cognitos certificate here? We'll probably use Cognito for other things as well down the line

Reply:
Oh shit, nice catch. Absolutely not.